src/components/TouchBar/FocusMenu/FocusMenu.jsx

```diff
@@ -53,73 +69,55 @@ class FocusMenu extends Component {
         (<FocusButton
           key={node.identifier}
           identifier={node.identifier}
-          active={this.state.origin}
-          onChangeOrigin={origin => this.onChangeOrigin({ origin })}
-          onRefocus={this.triggerChanges}
-          descriptionFlyTo={this.props.applyFlyTo.Description}

```

Comment:
storyTouch version might need this?

Reply:
This functionality (changing origin/anchor), refocusing and applying fly to is intact (as far as I can see), just that they are not passed down through props in the same way as before where values were (unnecessarily) passed back and forth a lot more. Whether functionality has been lost would probably show better by test-running the branch.